lua/packer.lua

```diff
@@ -13,6 +13,7 @@ local config_defaults = {
   snapshot_path = join_paths(stdpath 'cache', 'packer.nvim'),
   package_root = join_paths(stdpath 'data', 'site', 'pack'),
   compile_path = join_paths(stdpath 'config', 'plugin', 'packer_compiled.lua'),
+  merge_duplicates = false,
```

Comment:
Nitpick: maybe this should be slightly more descriptive, e.g. `merge_duplicate_configs`? Or is that too verbose?

Reply:
Yup, will change that!